storybook/test/__snapshots__/index.js.snap

```diff
@@ -3444,6 +3444,54 @@ exports[`Storyshots Components|ScrollLock Default 1`] = `
 </div>
 `;
 
+exports[`Storyshots Components|SlotFill Default 1`] = `
+Array [
+  <h2>
+    Profile
+  </h2>,
+  <p>
+    Name: 
+    <span />
+  </p>,
+  <p>
+    Age: 
+    <span />
+  </p>,
+]
+`;
+
+exports[`Storyshots Components|SlotFill With Context 1`] = `
+Array [
+  <h2>
+    Profile
+  </h2>,
+  <p>
+    Name: 
+    <span />
+  </p>,
+  <p>
+    Age: 
+    <span />
+  </p>,
+]
+`;
+
+exports[`Storyshots Components|SlotFill With Fill Props 1`] = `
+Array [
+  <h2>
+    Profile
+  </h2>,
+  <p>
+    Name: 
+    <span />
+  </p>,
+  <p>
+    Age: 
+    <span />
+  </p>,
+]
+`;
+
```

Comment:
It looks like storyshots doesn't support React Portal? :(

Reply:
I wouldn't be surprised if you would have to mock it somehow. See also:

https://github.com/WordPress/gutenberg/blob/0f2ec15ed40192a165f79790effa8edaf6504597/storybook/test/index.js#L9-L27

Reply:
I tried to run old unit tests that exist for the slot component:

https://github.com/WordPress/gutenberg/blob/master/packages/components/src/slot-fill/test/slot.js

They all fail, so it might be an actual issue with the implementation. Well, many of those tests are for the case that doesn't bubble virtually, so it's hard to tell :)

Reply:
I moved `slot-fill2` into `slot-fill` and replaced the current `bubblesVirtually` implementation with it. I had to remove only one assertion that was testing the component instance, but the rest is working fine. I'll add more tests later.